packages/shared-component--featureCard/src/featureCard.html

```diff
@@ -131,6 +131,15 @@
         </span>
     </span>
     {% endif %}
+    {% if content.fields.squirclePicker == 'Member deal' %}
+      <span class="coop-c-featureCard__squircle__label">
+        <span class="coop-c-featureCard__squircle__small-text">Member deal</span>
+      </span>
+      <span class="coop-c-featureCard__squircle__media coop-u-flex coop-u-flex-centered" aria-hidden="true">
```

Comment:
Mind adding `focusable="false"` for IE as well? We did it for all the Membership SVGs.

More context: https://github.com/alphagov/govuk-frontend/pull/774

Reply:
In fact, should put this bit on the SVG itself